src/container-instance.class.mts

```diff
@@ -1274,7 +1274,7 @@ export class ContainerInstance implements Disposable {
        * One important note is that the dependency *itself* is never resolved in the container.
        */
       const parameters = serviceMetadata.dependencies.map(resolvable => ({
-        id: this.resolveTypeWrapper(resolvable.typeWrapper),
+        id: resolvable.typeWrapper.eagerType,
```

Comment:
The now-removed method featured additional error-checking, e.g. null checks.  This should either be reimplemented or added to changeset.